Arithmetic.pp

```diff
@@ -65,7 +66,10 @@
     ternary() ( ::times:: #multiplication expression() )?
 
 ternary:
-    term() ( ::div:: #division expression() )?
+    quaternary() ( ::div:: #division expression() )?
+
+quaternary:
+    term() ( ::pow:: #power expression() )?
```

Comment:
Since PHP5.6, we have [the `**` operator](http://php.net/manual/en/language.operators.arithmetic.php] with the following precedence: http://php.net/manual/en/language.operators.precedence.php. It is a right associative operator. Is it the case here?